simulator/tabletop.py

```diff
@@ -403,6 +57,9 @@ def __mouse_released(self, event):
       # Snap to the closest cube.
       if best_cube:
         selected_cube.snap(best_cube, best_side)
+    '''
+
+    selected_cube.snap_to_grid((200, 200), self.__cubes, offset = (100, 100))
```

Comment:
I think that the grid size should be defined as a constant. In fact, is there ever a case where we would want the grid size to be different from the cube size? For that matter, you should probably do the same with the offset. Is there any case where the offset will *not* be half the grid size?